src/main/target/TINYFISH/target.h

```diff
@@ -0,0 +1,122 @@
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define USB_VCP_ENABLED 1
+
+#define TARGET_BOARD_IDENTIFIER "tFSH" // http://fishpepper.de/projects/tinyFISH
+
+#define CONFIG_FASTLOOP_PREFERRED_ACC ACC_NONE
+
+#define LED0                    PC14
+#define LED1                    PC15
+
+#define BEEPER                  PB2
+//#define BEEPER_INVERTED
+
+#define USE_EXTI
+#define MPU_INT_EXTI PC13
+#define USE_MPU_DATA_READY_SIGNAL
+#define EXTI15_10_CALLBACK_HANDLER_COUNT 1 // MPU_INT
+
+#define MPU6000_SPI_INSTANCE    SPI1
+#define MPU6000_CS_PIN          PA4
+
+#define GYRO
+#define USE_GYRO_SPI_MPU6000
+#define GYRO_MPU6000_ALIGN      CW180_DEG_FLIP
+
+#define ACC
+#define USE_ACC_SPI_MPU6000
+#define ACC_MPU6000_ALIGN       CW180_DEG_FLIP
+
+
+#if USB_VCP_ENABLED
+  #define USE_VCP
+  #define USB_IO
+  #define SERIAL_PORT_COUNT 4
+#else
+  #define SERIAL_PORT_COUNT 3
+#endif
+
+#define USE_UART1
+#define USE_UART2
+#define USE_UART3
+
+#define UART1_TX_PIN            PB6 
+#define UART1_RX_PIN            PB7 
+
+#define UART2_TX_PIN            PA14 // PA14 / SWCLK
+#define UART2_RX_PIN            PA15
+
+#define UART3_TX_PIN            PB10 // PB10 (AF7)
+#define UART3_RX_PIN            PB11 // PB11 (AF7)
+
+#define USE_SPI
+#define USE_SPI_DEVICE_1
+
+#define SPI1_NSS_PIN            PA4
+#define SPI1_SCK_PIN            PA5
+#define SPI1_MISO_PIN           PA6
+#define SPI1_MOSI_PIN           PA7
+
+#define USE_SPI
+#define USE_SPI_DEVICE_2 // PB12,13,14,15 on AF5
+
+#define SPI2_NSS_PIN            PB12
+#define SPI2_SCK_PIN            PB13
+#define SPI2_MISO_PIN           PB14
+#define SPI2_MOSI_PIN           PB15
+
+#define M25P16_CS_PIN           SPI2_NSS_PIN
+#define M25P16_SPI_INSTANCE     SPI2
+
+#define USE_FLASHFS
+#define USE_FLASH_M25P16
+#define ENABLE_BLACKBOX_LOGGING_ON_SPIFLASH_BY_DEFAULT
+
+#define USE_ADC
+#define BOARD_HAS_VOLTAGE_DIVIDER
+#define VBAT_ADC_PIN                PB1
+#define CURRENT_METER_ADC_PIN       PB0
+#define ADC_INSTANCE                ADC3
+#define VBAT_SCALE_DEFAULT          100
+
+
+#define LED_STRIP
+#define WS2811_PIN                      PA8
+#define WS2811_TIMER                    TIM1
+#define WS2811_DMA_CHANNEL              DMA1_Channel2
+#define WS2811_IRQ                      DMA1_Channel2_IRQn
+#define WS2811_DMA_TC_FLAG              DMA1_FLAG_TC2
+#define WS2811_DMA_HANDLER_IDENTIFER    DMA1_CH2_HANDLER
+
+#define DEFAULT_RX_FEATURE      FEATURE_RX_SERIAL
```

Comment:
I likely have the required scaling for the ACS712 already in there:

https://github.com/betaflight/betaflight/blob/master/src/main/target/ALIENFLIGHTF4/config.c#L54-L55